### ISSUE TITLE
Black icon when selected (Elementary OS)

### ISSUE THREAD

**Issue by daniloluca** [snwh/paper-gtk-theme#416]
Like you can see in the image, the icon become black when selected in file explorer on Elementary OS.
[With theme](https://www.dropbox.com/s/evgk898tlxxf6eh/with-theme.png?dl=0)

I do check, and this is not a problem from the icon theme. I remove the gtk theme and the problem stop happen, like you can see in this image:
[Without theme](https://www.dropbox.com/s/prgefpskmpl45zk/without-theme.png?dl=0)

I also check, and it's not because of the dark theme mode of the Elementary OS, it's happens in normal mode (light) too.

**Comment by Stidro** [comment]
Same problem! I'm using Ubuntu Gnome 16.04 with Pantheon-files. I'm using paper icons, theme, etc.

**Comment by Fuermohao** [comment]
it works very terrible on unity7.5.
Why the pictures you showed on your own website looks so beautiful,but on my desktop is just like shit.

**Comment by daniloluca** [comment]
I'm using Elementary OS, it uses Gala WM in the Pantheon DE.
I never tried it in Unity, but in here [Paper Theme](https://snwh.org/paper/theme) it seems ok.

**Comment by Fuermohao** [comment]
sorry,bro.I have tried eos today, the paper theme you developed is pretty good.And I really want to change the  window's color,could you  please tell me which css file and which part of the file should I change?

**Comment by daniloluca** [comment]
Sorry but I didn't developed the theme, the imagens are about an issue that I found in EOS.
I can tell you how I install the theme. 
I just did this:
sudo add-apt-repository ppa:snwh/pulp
sudo apt-get update
sudo apt-get install paper-icon-theme
sudo apt-get install paper-gtk-theme
Then I simply changed the icons and the GTK theme in the tweak tool.

**Comment by Fuermohao** [comment]
alright,but I already know how to install.thx all the same.

2016年11月18日 上午2:16于 Danilo de Lucas <notifications@github.com>写道：

Sorry but I didn't developed the theme, the imagens are about an issue that I found in EOS.
I can tell you how I install the theme.
I just did this:
sudo add-apt-repository ppa:snwh/pulp
sudo apt-get update
sudo apt-get install paper-icon-theme
sudo apt-get install paper-gtk-theme
Then I simply changed the icons and the GTK theme in the tweak tool.

**Comment by daniloluca** [comment]
Are you using Elementary OS?
Try turn the option "prefer dark variant" to true in Elementary's tweak tool. I think that's what you want.
Do you have the tweak tool installed?

**Comment by Fuermohao** [comment]
of course, I know how to deal with these things.I just want to use a theme that I really like.I am learning how to develop android apps. so I do prefer the material design.I have found the themes named Ashes,but it doesn't work perfect on gnome and unity, then I found the paper theme, it is not perfect because of the default color of the window's header bar looks like shit,though it have sevral kinds of colors for some default applications.More importantly, paper theme is not really material designed.So I spent all my yesterday to change the color and styles ( based on flat-pat theme) I really want.I am going to share it on my github these days.maybe also I'd like to develope a tool for flat-theme that can easily change the color to you wanted.
